src/markdown-pages/add-ons/rook.md

```diff
@@ -60,29 +70,25 @@ Additionally, `blockDeviceFilter` instructs Rook to use only block devices that
 For more information about the available options, see [Advanced Install Options](#advanced-install-options) above.
 
 The Rook add-on waits for the dedicated disk that you attached to your node before continuing with installation.
-If you attached a disk to your node, but the installer is waiting at the Rook add-on installation step, see [OSD pods are not created on my devices](https://rook.io/docs/rook/v1.0/ceph-common-issues.html#osd-pods-are-not-created-on-my-devices) in the Rook documentation for troubleshooting information.
+If you attached a disk to your node, but the installer is waiting at the Rook add-on installation step, see [OSD pods are not created on my devices](https://rook.io/docs/rook/v1.10/Troubleshooting/ceph-common-issues/#osd-pods-are-not-created-on-my-devices) in the Rook documentation for troubleshooting information.
 
 ## Filesystem Storage
 
 By default, for Rook versions earlier than 1.4.3, the cluster uses the filesystem for Rook storage.
 However, block storage is recommended for Rook in production clusters.
 For more information, see [Block Storage](#block-storage) above.
 
-When using the filesystem for storage, each node in the cluster has a single OSD backed by a directory in `/opt/replicated/rook`.
-Nodes with a Ceph Monitor also use `/var/lib/rook`.
-
-Sufficient disk space must be available to `/var/lib/rook` for the Ceph Monitors and other configs. For disk requirements, see [Add-on Directory Disk Space Requirements](/docs/install-with-kurl/system-requirements/#add-on-directory-disk-space-requirements). 
-
-We recommend a separate partition to prevent a disruption in Ceph's operation as a result of `/var` or the root partition running out of space.
+When using the filesystem for storage, each node in the cluster has a single OSD backed by a directory in `/opt/replicated/rook/`.
+We recommend a separate partition at `/opt/replicated/rook/` to prevent a disruption in Ceph's operation as a result the root partition running out of space.
```

Comment:
@chris-sanders has been pretty anti-partition in the past. I wonder if we want to be less prescriptive here and let them make their own decision on if they want to partition?

Reply:
I'll wait for more input here before making this change.

Reply:
I'd rather have a "we recommend not using this version of rook" tbh

Reply:
Not making this edit directly because it sounds like there's a chance this language changes anyway, but just noting that there's a typo in the last phrase: "...to prevent a disruption in Ceph's operation as a result **of (?)** the root partition running out of space."